Makefile

```diff
@@ -4,7 +4,7 @@
 #   See README.txt
 #   For qhulltest of the C++ interface, use Qt project file at src/qhull-all.pro
 #   For static builds, a simple alternative is src/libqhull_r/Makefile
-#   
+#
```

Comment:
Sorry for the whitespace changes — my editor deletes trailing whitespace by default.  Note that it is possible to [view the diff ignoring whitespace](https://stackoverflow.com/questions/37007300/how-to-ignore-whitespace-in-github-when-comparing) in various ways on github, e.g.  by appending `?w=1` to the URL.